src/Service/Organization/PackageManager.php

```diff
@@ -11,7 +11,7 @@
 use Munus\Control\Option;
 use Symfony\Component\Filesystem\Filesystem;
 
-final class PackageManager
+class PackageManager
```

Comment:
I'm mocking this class in unit tests

Reply:
And that's what worries me, that is, this class is doing too much and/or the interface and default implementation are missing.